### ISSUE TITLE
Doesn't work with dependent accounts

### ISSUE THREAD

**Issue by bdonnell** [gagebenne/pydexcom#32]
When you have an under 18 user, Dexcom requires setting up a dependent account under an adult account.  The dependent account gets its own login name and password according to Dexcom, but it doesn't work anywhere, even on Dexcom's own products.  Instead, you login with the main account, and then it will show a list of profiles, the first is the main account, and the following are the dependent accounts.  You pick the dependent account and then G6/Clarity/etc will continue with that dependent account and work great.

This lib takes the main account login info, but then there's no way to select the profile, so no data is ever returned.

**Comment by bdonnell** [comment]
Doing a little debugging on my part and I found something that may be helpful.

When logging in via the web uam1.dexcom.com it redirects to:
https://uam1.dexcom.com/multiaccount?loginOptions=<Base64 encoded JSON string>

The Base64 encoded JSON string decodes to:
{"main_user_accountname":"account_id_1", "dependent_1_accountname":"account_id_2", "dependent_2_accountname":"account_id_3"}

When I login with pydexcom and add some debug statements the account id returned from endpoint1 is the same as account_id_1 from the JSON string above.  I did try hard coding the account_id to the dependent account id I care about, and I set the password for the main account and the dependent accounts to the same thing, but pydexcom then still returns invalid password.  So there is more to it.
